builds.sh

```diff
@@ -19,13 +19,26 @@ shell="$(jq -r '
 ' "$json")"
 eval "$shell"
 
+_resolveRemoteArch() {
+	local img="$1"; shift
+
+	local arches
+	if ! arches="$(bashbrew remote arches --json "$img" 2>/dev/null)"; then # TODO somehow differentiate errors like 404 / 403, "insufficient_scope" from other bashbrew errors here so we can stop eating stderr
+		return
+	fi
+
+	echo "$arches"
+}
+
 # resolve an image reference to an architecture-specific imageId (digest of the image manifest)
 _resolve() {
 	local img="$1"; shift
 	local arch="$1"; shift
+	local bashbrewRemoteArches=
 
-	local bashbrewRemoteArches
-	if ! bashbrewRemoteArches="$(bashbrew remote arches --json "$img" 2>/dev/null)"; then # TODO somehow differentiate errors like 404 / 403, "insufficient_scope" from other bashbrew errors here so we can stop eating stderr
+	if [ -n "${1:+x}" ]; then
```

Comment:
This looks like the resolved architectures could be null/empty.

Reply:
Yes